src/test/java/seedu/address/logic/commands/CloseClaimCommandTest.java

```diff
@@ -0,0 +1,85 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.fail;
+import static seedu.address.logic.commands.CommandTestUtil.assertInsuranceCommandSuccess;
+import static seedu.address.testutil.Assert.assertThrows;
+import static seedu.address.testutil.TypicalClients.getTypicalAddressBook;
+import static seedu.address.testutil.TypicalIndexes.INDEX_THIRD_CLIENT;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.client.Client;
+import seedu.address.model.client.exceptions.ClaimException;
+import seedu.address.model.client.exceptions.InsurancePlanException;
+import seedu.address.model.client.insurance.InsurancePlan;
+import seedu.address.model.client.insurance.InsurancePlansManager;
+import seedu.address.model.client.insurance.InvalidInsurancePlan;
+import seedu.address.model.client.insurance.claim.Claim;
+import seedu.address.model.client.insurance.claim.ClaimStub;
+
+class CloseClaimCommandTest {
+    private Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+
+    @Test
+    void execute_validIndexUnfilteredList_success() throws CommandException,
+            InsurancePlanException, ClaimException {
+        Client clientToEdit = model.getFilteredClientList().get(INDEX_THIRD_CLIENT.getZeroBased());
+        InsurancePlansManager originalInsurancePlanManager = clientToEdit.getInsurancePlansManager();
```

Comment:
Naming would be better to follow class name. Hence `originalInsurancePlansManager` would be a better name (Note the 's')

Reply:
noted, good catch!